src/lando/main/models/landing_job.py

```diff
@@ -47,6 +47,14 @@ class LandingJob(BaseJob):
         Revision, through="RevisionLandingJob", related_name="landing_jobs"
     )
 
+    def is_pull_request_job(self) -> bool:
+        """Return True if the landing job has one revision that has a pull_number set."""
+        # TODO: RE: stack support, this will need to be modified.
+        return (
+            self.revisions.count() == 1
```

Comment:
nit: The revision count may throw us off in the future. I think the fact that `pull_number is not None` should be sufficient